base/file.jl

```diff
@@ -88,19 +88,30 @@ end
 mkdir(path::AbstractString, mode::Signed) = throw(ArgumentError("mode must be an unsigned integer; try 0o$mode"))
 mkpath(path::AbstractString, mode::Signed) = throw(ArgumentError("mode must be an unsigned integer; try 0o$mode"))
 
-function rm(path::AbstractString; recursive::Bool=false)
+function rm(path::AbstractString; force::Bool=false, recursive::Bool=false)
     if islink(path) || !isdir(path)
         @windows_only if (filemode(path) & 0o222) == 0; chmod(path, 0o777); end # is writable on windows actually means "is deletable"
-        unlink(path)
+        try
+            unlink(path)
+        catch err
+            if isa(err, UVError)
+                if force && err.prefix=="unlink" && err.code==Base.UV_ENOENT
+                    return
+                end
+                # convert UVError to SystemError
+                rethrow(SystemError(err.prefix, err.code, path))
```

Comment:
I don't know. The actual `unlink` system call occurred elsewhere, hence the `rethrow`. I'd be happy either way.